app/routes.py

```diff
@@ -0,0 +1,38 @@
+from app import app
+from app.blacklist import blacklist
+from app.controllers import UserController
+from flask import request, jsonify
+from flask_jwt_extended import *
+
+
+@app.route('/')
+def index():
+    return 'hello world'
+
+@app.route('/user', methods=['GET'])
+@jwt_required()
+def users():
+    user = get_jwt_identity()
```

Comment:
what is this `user` variable for?
I don't see it being used in this function.

Reply:
done